Formula/htop.rb

```diff
@@ -25,9 +25,15 @@ class Htop < Formula
   depends_on "python@3.9" => :build
   depends_on "ncurses" # enables mouse scroll
 
+  on_linux do
+    depends_on "lm-sensors"
+  end
+
   def install
     system "./autogen.sh"
-    system "./configure", "--prefix=#{prefix}"
+    args = "--prefix=#{prefix}"
```

Comment:
```suggestion
    args = ["--prefix=#{prefix}"]
```
You want this to be an array. In the current form, you have
```ruby
args = "--prefix=#{prefix}--enable-sensors"
```
so that's [where things are getting installed](https://github.com/Homebrew/homebrew-core/runs/2969684517?check_suite_focus=true#step:8:382):
```
  /usr/bin/install -c htop '/home/linuxbrew/.linuxbrew/Cellar/htop/3.0.5--enable-sensors/bin'
```